client/lib/plans/features-list.js

```diff
@@ -469,7 +469,7 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_REPUBLICIZE ]: {
 		getSlug: () => constants.FEATURE_REPUBLICIZE,
-		getTitle: () => i18n.translate( 'Advanced Social Media' ),
+		getTitle: () => i18n.translate( 'Advanced social media' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.